### ISSUE TITLE
Instructions are out of date

### ISSUE THREAD

**Issue by NZVengeance** [Baptiewright/streamdeck-nanoleaf#1]

**Comment by Baptiewright** [comment]
Yes they are, the plugin was rewritten to be part of the official StreamDeck "More Actions" list and to be honest I forgot there was even a Git for it.  I'll see if I can fix the bugs first then rewrite the docs.
